src/ts/layouts/ExtrinsicLayout.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { Component } from 'react'
+import { connect } from 'react-redux'
+import { IAppState } from '../background/store/all'
+import { LayoutContainer } from '../components/basic-components'
+import TopMenu from '../components/dashboard/TopMenu'
+import BottomMenu from '../components/dashboard/BottomMenu'
+import AccountDropdown from '../components/account/AccountDropdown'
+
+interface IDashboardProps extends StateProps {}
+
+class DashboardLayout extends Component<IDashboardProps> {
```

Comment:
Do you want to call it extrinsicLayout instead?

Reply:
I think I would just go with transferLayout.